src/GeositeFramework/js/Plugin.js

```diff
@@ -47,7 +47,43 @@ require(['use!Geosite',
                         downloadAsCsv: requestCsvDownload,
                         downloadAsPlainText: requestTextDownload,
                         dispatcher: N.app.dispatcher,
-                        suppressHelpOnStartup: _.partial(suppressHelpOnStartup, model)
+                        suppressHelpOnStartup: _.partial(suppressHelpOnStartup, model),
+                        resize: {
```

Comment:
The rest of the `app` object properties are references or function executions, can you do the same for `resize` so that it doesn't obscure the remaining arguments to `initialize`?